src/js/burials/labels.jsx

```diff
@@ -14,8 +14,8 @@ export const locationOfDeathLabels = {
 };
 
 export const allowanceLabels = {
-  nonService: 'Non-service-connected death',
-  service: 'Service-connected death',
+  nonService: 'Non-service-connected death (for Veteran receiving a pension or disability compensation)',
+  service: 'Service-connected death (for Veteran death related to, or a result of, a service-connected disability)',
```

Comment:
@cvalarida Hi, thanks, for checking. We don't need to add "as" here, but we should change "a" to "the".  I've updated to this: 
for Veteran death related to, or the result of, a service-connected disability.